go.mod

```diff
@@ -12,14 +12,14 @@ require (
 	github.com/activecm/mgosec v0.1.1
 	github.com/activecm/rita-bl v0.0.0-20200806232046-0db4a39fcf49
 	github.com/blang/semver v3.5.1+incompatible
+	github.com/briandowns/spinner v1.16.0
```

Comment:
While running the query which gathers the hostnames associated with the newly parsed in host IPs, we display a busy spinner.